src/DomBuilder/CustomerCreditTransferDomBuilder.php

```diff
@@ -77,19 +77,26 @@ public function visitPaymentInformation(PaymentInformation $paymentInformation):
         $serviceLevel = $this->createElement('SvcLvl');
         $serviceLevel->appendChild($this->createElement('Cd', 'SEPA'));
         $paymentTypeInformation->appendChild($serviceLevel);
+
+        if ($paymentInformation->getLocalInstrumentCode() || $paymentInformation->getLocalInstrumentProprietary()) {
+            $localInstrument = $this->createElement('LclInstrm');
+            if ($paymentInformation->getLocalInstrumentCode()) {
+                $localInstrument->appendChild($this->createElement('Cd', $paymentInformation->getLocalInstrumentCode()));
+            }
+
+            if ($paymentInformation->getLocalInstrumentProprietary()) {
```

Comment:
After some digging in the XSDs `LclInstrm` is a choise type. 

It is defined as:

```
    <xs:complexType name="LocalInstrument2Choice">
        <xs:choice>
            <xs:element name="Cd" type="ExternalLocalInstrument1Code"/>
            <xs:element name="Prtry" type="Max35Text"/>
        </xs:choice>
    </xs:complexType>
```

So it is one of the two and not both. Either `Cd` or `Prtry`.